src/v/storage/log_manager.cc

```diff
@@ -401,6 +402,71 @@ ss::future<> log_manager::remove(model::ntp ntp) {
     });
 }
 
+ss::future<> log_manager::remove_orphan(
+  ss::sstring data_directory_path, model::ntp ntp, model::revision_id rev) {
+    vlog(stlog.info, "Asked to remove orphan for: {} revision: {}", ntp, rev);
```

Comment:
nit: can we be more specific here i.e. `Asked to remove orphaned partition directory`

Reply:
I think no, because we remove all directories with rev less than provided.
We will log exact directories later